.github/workflows/release.yml

```diff
@@ -26,4 +26,4 @@ jobs:
           charts_dir: appuio
           charts_repo_url: https://charts.appuio.ch
         env:
-          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          CR_TOKEN: "${{ secrets.RELEASE_PAT }}"
```

Comment:
What's this secret? Doesn't seem to be configured in the repo

Reply:
You're right. I first need someone who has the vshnbot credentials :)
The GitHub token is insufficient as it doesn't trigger the changelog workflow ("recursive" workflow)